underfs/s3a/src/main/java/alluxio/underfs/s3a/S3AUnderFileSystem.java

```diff
@@ -593,11 +570,7 @@ protected String getRootKey() {
 
   @Override
   protected InputStream openObject(String key, OpenOptions options,
-      RetryPolicy retryPolicy) throws IOException {
-    try {
-      return new S3AInputStream(mBucketName, key, mClient, options.getOffset(), retryPolicy);
-    } catch (AmazonClientException e) {
```

Comment:
why is this changed?

Reply:
Because `S3AInputStream` constructor does not throw `AmazonClientException`, i remove the catch of exceptions.